controllers/front/iframe.php

```diff
@@ -58,10 +59,7 @@ public function postProcess()
         }
         if (!$authorized) {
             $this->errors[] =
-                $this->l('This payment method is not available.', 'validation');
-            if (!SaferPayConfig::isVersion17()) {
-                Tools::redirect($redirectLink);
-            }
+                $this->module->l('This payment method is not available.', 'validation');
```

Comment:
does this trans works? $this->module->l('This payment method is not available.', 'validation');